NEWS.md

```diff
@@ -1,3 +1,5 @@
+- `spinMyR()` gains new arguments `keepRmd` (default: `FALSE`) and `keepMd` (default: `TRUE`) (#15)
```

Comment:
Could you add a version title here, so it'll look like all the other news please